modules/understanding-upgrade-channels.adoc

```diff
@@ -7,159 +7,145 @@
+builds are available for early access to features, but updating to or from nightly
+builds is neither recommended nor supported. Nightly builds are not available in
 any upgrade channel.
 ====
 
 [discrete]
-== fast-{product-version} Channel
+== fast-{product-version} channel
 
-The fast-{product-version} channel is updated with new {product-version} patch
-versions as soon as Red Hat declares the given patch as a general availability
-release. As such, these releases are fully supported, are production quality and have
-performed well while available as a release candidate in the candidate-{product-version}
-channel from where they were promoted. Some time after a release appears in
-fast-{product-version} it will also appear in stable-{product-version}. Releases will
-never appear in stable-{product-version} before they appear in fast-{product-version}.
+The `fast-{product-version}` channel is updated with new {product-version}
+versions as soon as Red Hat declares the given version as a general availability
+release. As such, these releases are fully supported, are production quality, and have
+performed well while available as a release candidate in the `candidate-{product-version}`
+channel from where they were promoted. Some time after a release appears in the
+`fast-{product-version}` channel, it is added to the `stable-{product-version}` channel. Releases
+never appear in the `stable-{product-version}` channel before they appear in the `fast-{product-version}` channel.
 
-The fast-{product-version} channel also allows upgrading from a previous minor version of
-{product-title}.
+By subscribing to the `fast-{product-version}` channel, you have the ability to upgrade from a previous minor version of {product-title}.
 
 [discrete]
-== stable-{product-version} Channel
+== stable-{product-version} channel
 
-Like the fast-{product-version} channel, the stable-{product-version} channel will
-only contain releases that have been declared general availability are therefore
-fully supported. However the stable-{product-version} channel will gradually roll out
-releases to customers based on data from our SRE teams, support services, and
+Like the `fast-{product-version}` channel, the `stable-{product-version}` channel
+contains only releases that have general availability and are
+fully supported. While the `fast-{product-version}` channel contains releases as soon
+as they are released, releases are added to the the `stable-{product-version}` channel
+after data from Red Hat SRE teams, Red Hat support services, and
 pre-production and production environments that participate in our connected customer
-program, rather than being immediately available as they are in the fast-{product-version}
-channel. For patch and CVE fixes this delay can range from several hours to a day
+program confirm that they are stable. This delay can range from several hours to a day
```

Comment:
Stable confirmation does not impact promotion into the stable channel.  Current policy is that if a release goes into fast, it will go into the corresponding stable channel within at least a day or two, even if it suffers from critical bugs.  The impact of serious bugs is that we remove incoming update recommendations.  I think the previous master wording was closer to this behavior.  Can we make smaller pivots to address any style issues with the master language?

Reply:
> Would something like this be better?

Yeah, that proposal looks good to me.